x-pack/solutions/observability/plugins/apm/public/components/app/dependencies_inventory/dependencies_inventory_table/index.tsx

```diff
@@ -212,9 +214,7 @@ export function DependenciesInventoryTable() {
     ]
   );
   const showRandomSamplerBadge = data?.sampled && status === FETCH_STATUS.SUCCESS;
-  const fetchingStatus =
-    isPending(status) || isPending(timeseriesStatus) ? FETCH_STATUS.LOADING : FETCH_STATUS.SUCCESS;
-
+  const fetchingStatus =status;
```

Comment:
You can now rollback to what it was before, right?
```suggestion
const fetchingStatus =
    isPending(status) || isPending(timeseriesStatus) ? FETCH_STATUS.LOADING : FETCH_STATUS.SUCCESS;
```